### DISCUSSION TITLE
Johnsen ab 15.02.2022

### DISCUSSION THREAD

**Discussion by ruiyan-yu** [Digitaler-Impfnachweis/certification-apis#222] [Q&A]
Hallo,

Die KBV und die KVen haben am 28.01.2022 ein EBM-Update (Einheitlicher Bewertungsmaßstab) veröffentlicht.
Die Gebührenordnungspositionen (GOP) 88334, 88334I, 88334Y für Johnsen werden zum 31.01.2022 gestrichen.
Die GOPen 88334A, 88334B, 88334G, 88334H, 88334V und 8334W wurden aufgenommen mit der Gültigkeit ab 15.02.2022

Suffix A,G,V: Erstimpfung (Normal, Indikation Pflegeheim, berufliche Indikation)
Suffix B,H,W: Abschlussimpfung (Normal, Indikation Pflegeheim, berufliche Indikation)
=> 2-dosigen Grundimmunisierungsimpfung für Johnsen ab 15.02.2022

Gelten ab 15.02.2022 für Johnsen ebenfalls 1/2, 2/2, (2+x)/(2+x)?
Rezertifizierung für Novavax haben wir schon weggeschickt.
Werden wir bald wieder neue Änderungsverlangen von BMG bekommen?

Mit freundlichen Grüßen

Duria e.G

**Comment by mge-medisoftware** [comment]
Hallo,
technisch sind die neuen  Gebührenordnungspositionen (GOP) sogar schon ab sofort gültig.
So langsam wird es kompliziert/unübersichtlich, zumal es noch die "Empfehlung zur Optimierung der Grundimmunisierung nach 1-maliger Impfung mit der COVID-19 Vaccine Janssen" nach RKI/STIKO gibt.

"Von der Europäischen Arzneimittelbehörde (EMA) wurde im Dezember 2021 eine zweite Impfstoffdosis der COVID-19 Vaccine Janssen zur Zweitimpfung zugelassen. Die STIKO empfiehlt jedoch weiterhin allen Personen ≥ 18 Jahre, die eine erste Impfstoffdosis der COVID-19 Vaccine Janssen erhalten haben, ihre Grundimmunisierung mit einer zweiten Impfstoffdosis mit einem mRNA-Impfstoff zu optimieren. "

Ist das dann auch 2/2 oder eher eine Auffrischung wie bisher 2/1?

Viele Grüße aus Kiel
